lib/openxml/builder.rb

```diff
@@ -4,41 +4,72 @@
 #
 # This class mimics the XML Builder DSL.
 require "ox"
+require "ox_ext/node"
+require "ox_ext/hasattrs"
 
 module OpenXml
   class Builder
+    attr_reader :parent
 
-    def initialize
-      @document = Ox::Document.new(version: "1.0")
-      @current = @document
+    def initialize(standalone = nil)
+      @document = Ox::Document.new(version: "1.0", encoding: "utf-8", standalone: standalone)
+      @parent = @document
       yield self if block_given?
     end
 
     def to_s
-      Ox.dump @document
+      Ox.dump @document, with_xml: true, encoding: "UTF-8"
```

Comment:
Do we always want to pass `with_xml: true`? or only when we want `<?xml version="1.0" encoding="utf-8" standalone="yes"?>` prepended to the document?